totalRP3/Core/Utils.lua

```diff
@@ -528,6 +528,8 @@ local function GenerateLinkFormatter(line, defaultLinkColor, includeBraces, isMa
 			text = "[" .. text .. "]";
 		end
 
+		url = Utils.str.sanitize(url, false);
```

Comment:
I'd like to see the bool assigned to a variable with an actual name, because I can't remember off the top of my head what it's doing.

Something about multiline right? I'm thinking of people using links for hover tooltips in their about page.

Reply:
It's multiline, yeah.